src/start/RealmScreen.js

```diff
@@ -35,9 +33,21 @@ class RealmScreen extends PureComponent<Props, State> {
     error: undefined,
   };
 
+  isURL = str => {
```

Comment:
This did not need to be a class function. It would have been added to `/utils/url.js`.
Also to repeat, even though we rarely want to copy-paste code from StackOverflow, when we do we should include a link to the relevant page.